week03/css/about.css

```
@@ -0,0 +1,127 @@
+@font-face {
+    font-family: '국립박물관문화재단클래식B';
+    src: url('https://cdn.jsdelivr.net/gh/projectnoonnu/noonfonts_twelve@1.0/국립박물관문화재단클래식B.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: '국립박물관문화재단클래식M';
+    src: url('https://cdn.jsdelivr.net/gh/projectnoonnu/noonfonts_twelve@1.0/국립박물관문화재단클래식M.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+}
+
+* {
+    box-sizing: border-box;
+}
+
+#header {
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    padding: 10px 15px;
+    background-color: white;
+    color: #6a75ca;
+    font-size: 18px;
+    border-bottom: 3px solid #B4C7E7;
+}
+
+body {
+    background-color:#E6EEFB;
+}
+
+.title {
+    font-family: '국립박물관문화재단클래식B';
+    font-size: 40px;
+    color: black;
+    justify-content: center;
+    text-align: center;
+    display: flex;
+    margin: 250px 0 0 0;
+}
+
+.navbar {
+    background-color: white;
+    height: 75px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+
+}
+
+.nav1 {
+    float: left;
+    width: 20%;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+}
+
+.logo img {
+    width: 170px;
+    height: 80px;
+}
+
+.nav2 {
+    float: left;
+    width: 50%;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+}
+
+.nav3 {
+    float: left;
+    width: 40%;
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+}
+
+.navbar a {
+    text-decoration: none;
+    color: gray;
+    font-weight: lighter;
+    font-size: 23px;
+}
+.navbar a:hover {
+    color: #0D51FE;
+}
+
+.navbar ul {
+    float: right;
+}
+
+.navbar li {
+    margin-left: 15px;
+    list-style-type: none;
+    float: left;
+    font-family: '국립박물관문화재단클래식B';
+}
+
+.navbar li:hover {
+    border-bottom: 2px solid #0D51FE;
+}
+
+.BLog {
+    display: inline-block;
+    margin: 100 0 0 350px;
+}
+
+table {
+    width: 40%;
+    height: 35%;
+    border: 0px;
+    font-size: 16pt;
+    justify-content: left;
+    margin: -350px 0 0 750px;
+}
+
+.th {
+    color:#0D51FE;
+    font-family: '국립박물관문화재단클래식B';
+    font-size: 20pt;
+}

```

width를 지정할 경우, Interest 부분이 불필요하게 줄바꿈 되는 문제가 있어, 해당 부분을 삭제하면 좋겠습니다.

```suggestion
body main .content table {
    border: 0px;
    font-size: 16pt;
}
body main .content table tr {}
body main .content table tr th {
    padding-right: 10px;
    font-family: '국립박물관문화재단클래식B';
    font-size: 25px;
    color:#0D51FE;
}
body main .content table tr td {}
```